### ISSUE TITLE
Content not listed visually coded as list content

### ISSUE THREAD

**Issue by araposo-tistatech** [department-of-veterans-affairs/caseflow#14348]
## Description
Many lists are used to stylize information when viewing cases. As an example, when viewing a case, the Veteran ID and View All cases elements are read as a list of 2 items. No information on this page visually appears as a list. This content is not listed visually and should not be coded as list content. (Not Resolved: 12/13/2019) Some appear to be ok but others are not lists, just make sure if they look like a list to label them as a list and if they don’t look like a list do not label them as a list.  (Not Resolved 3/26/2020)

## Acceptance criteria
- [ ] Items visually appearing as lists are labeled as lists
- [ ] Items not visually appearing as lists are not labeled as lists

## Background/context/resources
Not resolved during latest accessibility audit

**Comment by yoomlam** [comment]
_[what is this chart?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | 
2 | 
3 | ||||||
5 | 
8 | 
```

Survey how much needs fixing. If easy, make the fix. Fix as much within timebox.
Focus on Case Details page initially. Scan other pages: Search page, etc.
Timebox to 3.
